html2docx.py

```diff
@@ -10,6 +10,16 @@
 WHITESPACE_RE = re.compile(r"\s+")
 
 
+def get_attrs(attrs: List[Tuple[str, Optional[str]]]) -> List[str]:
```

Comment:
I find this function name very terse. It is named "get_attrs", accepts a argument "attrs" and returns a variable "args". I think it could be a bit more descriptive or have a docstring to describe its purpose.